v2/internal/pkg/release/cincinnati.go

```diff
@@ -245,7 +271,7 @@ func (o *CincinnatiSchema) GetReleaseReferenceImages(ctx context.Context) ([]v2a
 
 	imgs, err := o.Signature.GenerateReleaseSignatures(ctx, allImages)
 	if err != nil {
-		o.Log.Error("%v", err)
+		o.Log.Error("%w", err)
```

Comment:
I don't think the logger understands the error wrapping verb `%w`. But more importantly, we're logging the errors but not acting upon it. Don't we want to return an error here?

Reply:
Yep I''l add a return error